packages/store/ts/config/v2/input.ts

```diff
@@ -39,7 +38,6 @@ export type TableInput = {
   readonly name?: string;
   readonly schema: SchemaInput;
   readonly key: readonly string[];
-  readonly tableId?: Hex;
```

Comment:
i thought the idea with allowing a `tableId` input is that it could technically be overridden to an id that is not representable in UTF8? I guess that would break a lot of other assumptions, is that why you're removing it here?

Reply:
That was the intent but now that there's a namespace and name override and they're contextual, it's a little scary to allow overriding the `tableId` directly. I think it leads to hard to predict/understand behavior.

I'm not aware of any strings JS can't represent/store accurately. There's various encoding options for these too.

We could still add this option again in the future when we have a strong use case, but we'll wanna be sure to validate that the `tableId` matches the table type or namespace it's used within.